apps/backend/src/api/compose_deployment.py

```
@@ -6,27 +6,28 @@
 """
 
 import logging
-from typing import Optional
-from fastapi import APIRouter, Depends, HTTPException, Body
+from typing import Any
```

Ruff (0.12.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)